ros/kxr_controller/python/kxr_controller/kxr_interface.py

```diff
@@ -90,6 +102,49 @@ def servo_off(self, joint_names=None):
         goal.servo_on_states = [False] * len(joint_names)
         client.send_goal(goal)
 
+    def adjust_angle_vector(
+            self, joint_names=None, error_threshold=np.deg2rad(5)):
+        if joint_names is None:
+            joint_names = self.joint_names
+        goal = AdjustAngleVectorGoal()
+        client = self.adjust_angle_vector_client
+        if client.get_state() == actionlib_msgs.msg.GoalStatus.ACTIVE:
+            client.cancel_goal()
+            client.wait_for_result(timeout=rospy.Duration(10))
+        goal.joint_names = joint_names
+        if isinstance(error_threshold, numbers.Number):
+            error_threshold = [
+                error_threshold for _ in range(len(joint_names))]
+        goal.error_threshold = error_threshold
+        client.send_goal(goal)
+
+    def wait_interpolation(
+            self, *args,
+            timeout=0,
+            adjust_angle_vector=False,
+            joint_names=None,
+            error_threshold=np.deg2rad(5),
+            **kwargs):
+        if adjust_angle_vector is False:
+            return super(KXRROSRobotInterface, self).wait_interpolation(
+                *args, timeout=timeout, **kwargs)
+        if timeout == 0:
+            timeout = float('inf')
+        start_time = time.time()
```

Comment:
We should use rospy.Time()

Reply:
I add commit [`7c5e625` (#77)](https://github.com/iory/rcb4/pull/77/commits/7c5e62515960887b6613536d55167a203dffb7d6)